File: CommerceLink/develop/VSI.EDGEAXConnector.Data/ViewModels/XMLMap.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace VSI.EDGEAXConnector.Data.ViewModels
{
    public class XMLMap
    {
        public string SourceEntity { get; set; }
        public string Name { get; set; }
        public string Type { get; set; }
        public string XML { get; set; }
        public bool isActive { get; set; }
    }
}

File: CommerceLink/develop/VSI.EDGEAXConnector.WebUI/src/app/Entities/XMLMap.ts
export class XMLMap {
    public SourceEntity: string;
    public Name: string;
    public Type: string;
    public XML: string;
    public isActive: boolean;
    constructor() {
        this.SourceEntity = "";
        this.Name = "";
        this.Type = "";
        this.XML = "";
        this.isActive = true;
    }
}